Project1/Calculator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project1
+{
+    internal class Calculator
```

Comment:
Nazwałbym to jak już CalculatorBase, bo potem bedzie mozna dziedziczyć po tym przy advancedCalculator